src/plugins/index_pattern_field_editor/public/components/field_editor/field_editor.tsx

```diff
@@ -137,6 +138,11 @@ const geti18nTexts = (): {
   },
 });
 
+const changeWarning = i18n.translate('indexPatternFieldEditor.editor.form.changeWarning', {
```

Comment:
nit: this might be better inside the `geti18nTexts` handler above.

Reply:
The `geti18nTexts` returns `{ title: string; description: JSX.Element | string };` and this is a single string.

Reply:
Oh I missed that!